.github/workflows/build-deb.yml

```diff
@@ -0,0 +1,37 @@
+name: Build Debian Package
+on:
+  push:
+      tags: [ '[0-9]+.[0-9]+.[0-9]+' ]
+  pull_request:
+      tags: [ '[0-9]+.[0-9]+.[0-9]+' ]
+
+jobs:
+  package-ubuntu-20-amd64:
+    runs-on: ubuntu-20.04
```

Comment:
Anything against `ubuntu-latest` for this?

Reply:
No, you're right. Changed it